### PR TITLE
Integrate russian and arabic EULA and privacy notice

### PR DESCRIPTION
**Pull request by i336616**
Closes issues:
EXPOSUREAPP-1647
EXPOSUREAPP-1736

Added Russian & Arabic localized text for Usage & Privacy notices
Added right-to-left language support to HtmlTextView

### CODE REVIEW

**Comment by aleksandr-tikhonov** [comment]
https://jira.itc.sap.com/browse//EXPOSUREAPP-1736 doesn't exist for me

**Comment by aleksandr-tikhonov** [comment]
Russian text looks good for me

**Review by steins-code — APPROVED** [review]

**Review by aleksandr-tikhonov — APPROVED** [review]

File: src/xcode/ENA/ENA.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -424,6 +424,7 @@
 		130CB19B246D92F800ADE602 /* ENAUITestsOnboarding.swift */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.swift; path = ENAUITestsOnboarding.swift; sourceTree = "<group>"; };
 		13156CFE248C19D000AFC472 /* de */ = {isa = PBXFileReference; lastKnownFileType = text.html; name = de; path = de.lproj/usage.html; sourceTree = "<group>"; };
 		13156D00248CDECC00AFC472 /* en */ = {isa = PBXFileReference; lastKnownFileType = text.html; name = en; path = en.lproj/usage.html; sourceTree = "<group>"; };
+		1347D58924B7753C0028E102 /* ar */ = {isa = PBXFileReference; lastKnownFileType = text.html; name = ar; path = "ar.lproj/privacy-policy.html"; sourceTree = "<group>"; };
```
Comment (MichaelPFrey) [inline src/xcode/ENA/ENA.xcodeproj/project.pbxproj:427]: This looks odd.

There is a line added for `ar.lproj/privacy-policy.html`, but there isn't a line added for
```
ar.lproj/usage.html
ru.lproj/privacy-policy.html 
ru.lproj/usage.html 
```

Comment (steins-code) [reply src/xcode/ENA/ENA.xcodeproj/project.pbxproj:427]: @i336616 can you check whether some reference is missing? :)

Comment (i336616) [reply src/xcode/ENA/ENA.xcodeproj/project.pbxproj]: Re-added TR (turkish) localization references back into project.

**Pull request merged**